code/controllers/CMSPageAddController.php

```diff
@@ -85,8 +75,21 @@ public function AddForm() {
 						'Note: Some page types are not allowed for this selection'
 					)
 				)
+			),
+			$typeField = new OptionsetField(
+				"PageType",
+				DBField::create_field(
+					'HTMLFragment',
+					sprintf($numericLabelTmpl, 2, _t('CMSMain.ChoosePageType', 'Choose page type'))
+				),
+				$pageTypes,
+				'Page'
 			)
 		);
+
+		// TODO Insert page step title
```

Comment:
@tractorcow This was the code for the title I had working at one stage but no longer works, you think you can fix it?